keyboards/silverbullet44/keymaps/default/keymap.c

```diff
@@ -0,0 +1,200 @@
+ * |------+------+------+------+------+------|                                  |------+------+------+------+------+------|
+ * |      |      |Music |FrqUp |Sound |      |                                  | SPD+ | MOD+ | VAL+ | SAD+ | HUE+ |      |
+ * |------+------+------+------+------+------|                                  |------+------+------+------+------+------|
+ * |      |      |Scale |FrqDwn|      |      |-------.-------.  ,---------------| SPD- | SPD- | VAL- | SAD- | HUE- |      |
+ * `-----------------------------------------/       /       /   \       \       \----------------------------------------'
+ *                          |      |      | /       /       /     \       \       \  |      |      |
+ *                          |      |      |/       /       /       \       \       \ |      |      |
+ *                          `-----------------------------'         '------------------------------'
+*/
+
+  [_ADJUST] = LAYOUT(
+  _______, XXXXXXX, XXXXXXX, CK_RST,  XXXXXXX, XXXXXXX,                      XXXXXXX, XXXXXXX,  XXXXXXX, XXXXXXX, XXXXXXX, _______,
+  XXXXXXX, XXXXXXX, MU_TOG,  CK_UP,   AU_TOG,  XXXXXXX,                      RGB_SPI, RGB_MOD,  RGB_VAI, RGB_SAI, RGB_HUI, XXXXXXX,
+  XXXXXXX, XXXXXXX, MU_MOD,  CK_DOWN, XXXXXXX, XXXXXXX,                      RGB_SPD, RGB_RMOD, RGB_VAD, RGB_SAD, RGB_HUD, XXXXXXX,
+                             XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,    XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX
+  )
+};
+
+#ifdef RGBLIGHT_ENABLE
+// Following line allows macro to read current RGB settings
+extern rgblight_config_t rgblight_config;
+#endif
+
+int  RGB_current_mode;
+bool alt_pressed = false;
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+#ifdef RGBLIGHT_ENABLE
+        case RGBRST:
+            if (record->event.pressed) {
+                eeconfig_update_rgblight_default();
+                rgblight_enable();
+                RGB_current_mode = rgblight_config.mode;
+            }
+            break;
+#endif
+        case KC_SCLN:
+            if (keyboard_report->mods & MOD_BIT(KC_LSFT)) {
+                if (record->event.pressed) {
+                    unregister_code(KC_LSFT);
+                    tap_code16(keycode);
+                    register_code(KC_LSFT);
+                }
+                /*  } else if (keyboard_report->mods & MOD_BIT(KC_RSFT)) {
+                    if (record->event.pressed) {
+                      unregister_code(KC_RSFT);
+                      tap_code16(keycode);
+                      register_code(KC_RSFT);
+                    } */
+            } else {
+                if (record->event.pressed) {
+                    tap_code16(S(keycode));
+                }
+            }
+            return false;
+            break;
+        case KC_00:
+            if (record->event.pressed) {
+                SEND_STRING("00");
+            }
+            return false;
+            break;
```

Comment:
If you wanted to be tricky here, you could use: 
```c
        case KC_00:
            if (record->event.pressed) {
                tap_code(KC_0);
                register_code(KC_0);
            } else { 
                register_code(KC_0);
            }
            return return;
            break;
```

That way, if you tap it, it will do `00`, but if you hold it, it will repeat the 0's, per the OS's setting.s

Reply:
I see!
I wanna be tricky!!